### PR TITLE
[Goodie] Inkscape cheat sheet

### PR DESCRIPTION
**Pull request by octomwm**
Hi everyone,

This is my first zci :) No surprise here, it's a basic cheat sheet dedicated to inkscape.

IA Page: https://duck.co/ia/view/inkscape_cheat_sheet

**What does your Instant Answer do?**
Gives you a basic inkscape cheat sheet.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://inkscape.org/en/doc/keys091.html

**Why did you choose this data source?**
It's the Inkscape doc, it's reliable, regularly updated, and is just the biggest inkscape hotkeys source.

**Are there any other alternative (better) data sources?**
Not that I know off :)

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Designers, graphist that don't have the money to use Illustrator :)

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
I've just suggested an idea, here is the link : https://duck.co/ideas/idea/6320/inkscape-cheat-sheet

**Are you having any problems? Do you need our help with anything?**
No.

**Where did you hear about DuckDuckHack? (For first time contributors)**
I was already in the ddg teams when ddh was created.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
Inkscape cheat sheet screenshot

### CODE REVIEW

**Comment by mintsoft** [comment]
Hi @octomwm thanks for the contribution, can you paste the cheatsheet pull request template into the description for us?  https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md  Thanks

**Comment by mintsoft** [comment]
@moollaza I can't find anything in the pipeline or a thread for this one

**Comment by mintsoft** [comment]
@octomwm Given that your PR isn't connected to an approved IA Idea we need to evaluate it first, but @moollaza will swing-in soon to do so no doubt!

**Comment by MariagraziaAlastra** [comment]
Hey @octomwm thanks for this!
@mintsoft @moollaza I think it's ok to approve it :)

Please @octomwm, could you create an idea on the forums? https://duck.co/ideas
And then please claim it, so that other people will know that you're working on it.

Thanks! ;)

**Comment by moollaza** [comment]
This one is good, @octomwm and I discussed this one over email. :+1:

**Comment by mintsoft** [comment]
@MariagraziaAlastra I've never used inkscape, are you more suitable to review this?

**Comment by MariagraziaAlastra** [comment]
@mintsoft Yeah, I've used it, I'll do the review :)

File: share/goodie/cheat_sheets/inkscape.json [hunk 1]
```diff
@@ -0,0 +1,277 @@
+{
+    "id": "inkscape_cheat_sheet", 
+    "name": "Inkscape",
+    "description": "Vector Graphics Editor",
```
Comment (MariagraziaAlastra) [inline share/goodie/cheat_sheets/inkscape.json]: @octomwm could you also add "keyboard" as the template type? It's the default anyways, but it's better to specify it :)

File: share/goodie/cheat_sheets/inkscape.json [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+            {
+                "key": "t",
+                "val": "Text tool"
+            },
+            {
+                "key": "a",
+                "val": "Spray tool"
+            },
+            {
+                "key": "E",
```
Comment (MariagraziaAlastra) [inline share/goodie/cheat_sheets/inkscape.json]: Is this [Shift] [E]?

**Comment by MariagraziaAlastra** [comment]
I approve the content, I made some observations but they're really minor.
@moollaza if you sign off on this I'd be happy to merge it right now :)

**Comment by moollaza** [comment]
@octomwm @MariagraziaAlastra LGTM :+1:

**Comment by MariagraziaAlastra** [comment]
Thanks @octomwm, will let you know when it's live!

**Comment by jagtalon** [comment]
@MariagraziaAlastra @moollaza @octomwm would mac users be confused with this since `[Ctrl] [D]` is `[⌘] [D]`?

**Comment by octomwm** [comment]
Hi everyone,
@mintsoft : I've just added the cheatsheet pull request template into the description, I hope it's ok :)
@MariagraziaAlastra : Thanks for the review, I made a suggestion on duck.co ( https://duck.co/ideas/idea/6320/inkscape-cheat-sheet ). Also you where right the E is Shift + E, I just made the change, and added the template to the json file :)
@Moozla : Great !

**Comment by MariagraziaAlastra** [comment]
Yeah, saw that! Thanks a lot @octomwm :)

**Comment by moollaza** [comment]
@octomwm congrats! It's live: https://duckduckgo.com/?q=inkscape+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**